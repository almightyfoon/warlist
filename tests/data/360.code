c8501b5veD4Q4M4H4H4H5gc9c95e5e5f___-0q4mcn4H4H5j5j5j5k4U4W4U4W4U4W