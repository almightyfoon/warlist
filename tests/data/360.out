https://conflictchamber.com/#c8501b5vmN4Q4M4H4H4H5gmVmV5e5e5f___-0q4mcn4H4H5j5j5j5k4U4W4U4W4U4W

Circle Army - 73 / 75 points

[Wurmwood 1] Wurmwood, Tree of Fate [+27]
 - Brennos the Elderhorn [15]
 - Megalith [20]
 - Pureblood Warpwolf [17]
 - Woldwyrd [9]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Wayfarer [4]
Gallows Grove [2]
Gallows Grove [2]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Shifting Stones [3]

Circle Army - 75 / 75 points
[Theme] The Wild Hunt

[Baldur 1] Baldur the Stonecleaver [+31]
 - Woldwrath [37]
 - Woldwyrd [9]
 - Woldwyrd [9]
War Wolf [2]
War Wolf [2]
War Wolf [2]
Wolf Lord Morraig [8]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [0(4)]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [0(4)]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]
